casanovo/data/ms_io.py

```
@@ -6,14 +6,69 @@
 import os
 import re
 from pathlib import Path
+from collections.abc import Iterator
 from typing import List
```

Ruff (0.15.6)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)